install-dev/controllers/http/database.php

```diff
@@ -226,6 +226,11 @@ public function display(): void
             $this->smtp_port = $this->session->smtp_port;
         }
 
+        $staticPrefix = defined('_PS_INSTALLER_STATIC_DB_PREFIX_') ? _PS_INSTALLER_STATIC_DB_PREFIX_ : null;
+        if (is_string($staticPrefix) && $staticPrefix !== '') {
```

Comment:
$staticPrefix = defined('_PS_INSTALLER_STATIC_DB_PREFIX_') ? _PS_INSTALLER_STATIC_DB_PREFIX_ : null;
        if (is_string($staticPrefix) && $staticPrefix !== '') {
```suggestion
        if (defined('_PS_INSTALLER_STATIC_DB_PREFIX_') && !empty(_PS_INSTALLER_STATIC_DB_PREFIX_)) {
```

maybe?

Reply:
that, I think, could work, but someone also could put `true` or `1` as a prefix 😅

Reply:
@kpodemski (string) true/1